### PR TITLE
Post/migration to rails 3.2

### PR DESCRIPTION
**Pull request by lubc**
This PR adds an article about how to upgrade from Rails `3.1` to `3.2`

Please check it out. Thanks!

### CODE REVIEW

File: _posts/2017-08-28-upgrade-to-rails-3.markdown [hunk 1]
```diff
@@ -174,4 +174,4 @@ lib/tasks/ombulabs_patches/s3_backup.rake
 ```
 
 <h2 id="next-steps">8. Next steps</h2>
-After you get your application propertly running in Rails 3.0, you will probably want to keep working on this Rails upgrade journey. So don't forget to check our complete [Rails upgrade series](https://www.ombulabs.com/blog/tags/upgrades) to make that easy.
+After you get your application properly running in Rails 3.0, you will probably want to keep working on this Rails upgrade journey. So don't forget to check our complete [Rails upgrade series](https://www.ombulabs.com/blog/tags/upgrades) to make that easy.
```
Comment (etagwerker) [inline _posts/2017-08-28-upgrade-to-rails-3.markdown]: 👍

File: _posts/2017-10-23-upgrade-to-rails-3-2.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Upgrade Rails from 3.1 to 3.2"
+date: 2017-10-23 10:00:00
```
Comment (etagwerker) [inline _posts/2017-10-23-upgrade-to-rails-3-2.md]: @lubc Once this post is approved please update this to the approval date and regenerate the HTML.

File: _posts/2017-10-23-upgrade-to-rails-3-2.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+categories: ["rails", "upgrades"]
+author: "luciano"
+---
+
+This is the third article of our [Upgrade Rails series](https://www.ombulabs.com/blog/tags/upgrades). We will be covering the most important aspects that you need to know to update your [Ruby on Rails](http://rubyonrails.org/) application from [version 3.1](http://guides.rubyonrails.org/3_1_release_notes.html) to 3.2.
```
Comment (etagwerker) [inline _posts/2017-10-23-upgrade-to-rails-3-2.md]: @lubc Please add a link to 3.2 as well, just like you did for 3.1

File: _posts/2017-10-23-upgrade-to-rails-3-2.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+<h2 id="tools">3. Tools</h2>
+Rails 3.2 comes with a [generator](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#the-update-task) that helps you to update the configuration files. The `rake rails:update` generator will identify every configuration file in your application that differs from a new Rails application. When it detects a conflict, it will offer to overwrite your file. Keep in mind that many of your files will be different because you’ve made changes from a default new Rails application. When the generator offers to overwrite a file, enter a `d` for review the differences.
```
Comment (etagwerker) [inline _posts/2017-10-23-upgrade-to-rails-3-2.md]: @lubc "for review the differences" change it to "in order to review the differences"

**Comment by lubc** [comment]
@etagwerker updated!

**Pull request merged**